crates/sui-adapter-transactional-tests/tests/shared/by_value_shared_object_deletion_via_make_move_vec_fails.move

```diff
@@ -34,37 +34,37 @@ module t2::o2 {
         transfer::public_share_object(o)
     }
 
-    public fun delete(o2: vector<Obj2>) {
+    public fun delete(mut o2: vector<Obj2>) {
         let o = vector::pop_back(&mut o2);
         deleter(o);
         vector::destroy_empty(o2);
     }
 
-    public fun freezee(o2: vector<Obj2>) {
+    public fun freezee(mut o2: vector<Obj2>) {
         let o = vector::pop_back(&mut o2);
         freezer(o);
         vector::destroy_empty(o2);
     }
 
-    public fun dof(parent: &mut Obj2, o2: vector<Obj2>) {
+    public fun dof_(parent: &mut Obj2, mut o2: vector<Obj2>) {
```

Comment:
You shouldn't have needed these renames. Did the migration script do this?

Reply:
I think I did it manually before the most recent naming PR -- I'll go back and see if I can revert the name changes